readme.md

```diff
@@ -287,6 +287,8 @@ If you're on Windows, do yourself a favor and use [`cmder`](http://cmder.net/) i
 
 ## Related
 
+- [chalkie](https://github.com/shawwn/chalkie) - Browser support for chalk
```

Comment:
This is misleading. When you say browser support people will assume you support printing to the console, not HTML. You should be explicit about what it does.